applications/callflow/src/module/cf_callflow.erl

```diff
@@ -32,11 +32,17 @@ handle(Data, Call) ->
 maybe_branch_callflow(Data, Call) ->
     Id = wh_json:get_value(<<"id">>, Data),
     case couch_mgr:open_doc(whapps_call:account_db(Call), Id) of
-        {'ok', JObj} ->
-            lager:info("branching to callflow ~s", [Id]),
-            Flow = wh_json:get_value(<<"flow">>, JObj, wh_json:new()),
-            cf_exe:branch(Flow, Call);
         {'error', R} ->
             lager:info("could not branch to callflow ~s, ~p", [Id, R]),
-            cf_exe:continue(Call)
+            cf_exe:continue(Call);
+        {'ok', JObj} ->
+            case whapps_call_command:channel_status(Call) of
```

Comment:
I think this needs to be `b_channel_status`. You'll only get the error tuple if the Call has no call_id set in it. b_channel_status will actually query and wait for the response.